src/handlers/uniswap/customEvents.ts

```diff
@@ -0,0 +1,132 @@
+
+                    if (eventData.sender === eventData.recipient){
+                        return [
+                            {
+                                address: eventData.recipient,
+                                title: 'Your swap is complete',
+                                message: `The swap you initiated has been completed. Open wallet to see more details.`,
+                                options: {}
+                            }
+                        ]
+                    }
+
+                    return [
+                        {
+                            address: eventData.sender,
+                            title: 'Your swap is complete',
+                            message: `The swap you initiated has been completed`,
+                            options: {}
+                        },
+                        {
+                            address: eventData.recipient,
+                            title: 'Received Swapped Tokens',
+                            message: `You have received tokens from a swap initiated by ${eventData.sender}. Open wallet for more details.`,
+                            options: {}
+                        }
+                    ]
+                }
+            },
+            {
+                eventName: 'BurnEvent',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as BurnEventData;
+                    return [
+                        {
+                            address: eventData.owner,
+                            title: 'Removed from liquidity position',
+                            message: `Your liquidity position has been removed and a total of ${eventData.amount / parseFloat("1000000000000000")} has been returned.`,
+                            options: {}
+                        }
+                    ]
+                }
+            },
+            {
+                eventName: 'MintEvent',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as MintEventData;
+                    if (eventData.owner === eventData.sender){
+                        return [
+                            {
+                                address: eventData.owner,
+                                title: 'Liquidity Minted',
+                                message: `Your liquidity of the amount ${eventData.amount / parseFloat("1000000000000000")} has been minted`,
+                                options: {}
+                            }
+                        ]
+                    }
+                    else{
+                        return [
+                            {
+                                address: eventData.owner,
+                                title: 'Liquidity minted for you',
```

Comment:
Is it possible to determine what Liquidity or Asset token is involved here?

Reply:
For that, the solution would be more involved. Currently, the event doesn't give that information out, probably because it wasn't intended for this such a use case. One solution is to fully monitor events through etherscan, which links to the transaction and then parse the tokens involved. But this solution as quite a lot of pot holes for reasons you can imagine.